internal/rpc/dial.go

```diff
@@ -89,7 +89,13 @@ func Dial(ctx context.Context, ctl *dbmodel.Controller, modelTag names.ModelTag,
 	for _, hps := range ctl.Addresses {
 		for _, hp := range hps {
 			if maybeReachable(hp.Scope) {
-				urls = append(urls, websocketURL(fmt.Sprintf("%s:%d", hp.Value, hp.Port), modelTag, finalPath))
+				var ip string
+				if hp.Type == string(network.IPv6Address) {
```

Comment:
suggestion: 
```
addr := net.UDPAddr{
  IP: net.ParseIP(hp.Value),
  Port: hp.Port,
}
ip := addr.String()
```

Reply:
the issue is that we have 3 types:
- ipv4
- ipv6
- hostname

Would this work for hostname?